docs/source/api/apollo-server.md

```diff
@@ -480,6 +480,17 @@ addMockFunctionsToSchema({
 
```

Comment:
while you're in this file, the `apiKey` section could use a reference to `APOLLO_KEY` (probably good to still mention ENGINE_API_KEY as an older name)